lib/sentry.dart

```diff
@@ -365,6 +365,11 @@ class Event {
   /// set in [SentryClient.userContext] for this logged event.
   final User userContext;
 
+  /// The context interfaces provide additional context data.
+  /// Typically this is data related to the current user,
+  /// the current HTTP request.
```

Comment:
Can we add a link to Sentry docs where this is explained? At this point it is very hard for me to page in all the knowledge about Sentry to know what's going on purely from code.